src/App/MainPage.xaml

```diff
@@ -37,123 +44,128 @@
-                                                </StackPanel>
-                                            </StackPanel>
-                                        </Flyout>
-                                    </Button.Flyout>
-                                </Button>
-                                <Button x:Name="SettingsButton" x:Uid="SettingsButton" Visibility="Visible" Padding="5" Margin="5,0,0,0">
-                                    <StackPanel Orientation="Horizontal">
-                                        <TextBlock FontFamily="Segoe MDL2 Assets" Text="&#xE713;" FontSize="30" VerticalAlignment="Center" />
-                                    </StackPanel>
-                                    <Button.Flyout>
-                                        <Flyout x:Name="SettingsFlyout" Opening="SettingsFlyout_Opening">
-                                            <StackPanel Orientation="Vertical" HorizontalAlignment="Center">
-                                                <TextBlock Style="{StaticResource FlyoutPickerTitleTextBlockStyle}" x:Uid="SettingsHeader" HorizontalAlignment="Center"/>
-                                                <HyperlinkButton x:Uid="SettingsSubHeader" NavigateUri="https://microsoft.github.io/FactoryOrchestrator/use-the-factory-orchestrator-app#app-settings" HorizontalAlignment="Center"/>
-                                                <StackPanel x:Name="SettingsStackPanel" Orientation="Vertical" HorizontalAlignment="Left">
-                                                    <CheckBox x:Name="SettingsTrackExecution" x:Uid="SettingsTrackExecution" IsThreeState="False" IsChecked="True" Checked="SettingsTrackExecution_Checked" Unchecked="SettingsTrackExecution_Checked" />
-                                                    <CheckBox x:Name="SettingsShowExternalTasks" x:Uid="SettingsShowExternalTasks" IsThreeState="False" IsChecked="True" Checked="SettingsShowExternalTasks_Checked" Unchecked="SettingsShowExternalTasks_Checked" />
-                                                </StackPanel>
+
+                        <StackPanel x:Name="ButtonPanel" Orientation="Horizontal" HorizontalAlignment="Right">
+
+                            <Button x:Name="ConfigButton" Background="{ThemeResource ControlFillColorSecondary}" x:Uid="ConfigButton" Visibility="Visible" Padding="5" Margin="0">
+                                <StackPanel Orientation="Horizontal">
+                                    <!--<TextBlock FontFamily="Segoe MDL2 Assets" Text="&#xEDA3;" FontSize="30" VerticalAlignment="Center" />-->
+
+                                    <SymbolIcon Symbol="More" Foreground="{ThemeResource TextFillColorSecondary}" Width="30" Height="30"/>
+                                </StackPanel>
+                                <Button.Flyout>
+                                    <Flyout x:Name="ConfigFlyout" Opening="ConfigFlyout_Opening">
```

Comment:
if this is generic sysinfo, what else would be useful?

- battery info?
- total Memory?
- free disk space / used space?
- Serial Number
- Computer Hardware ID
- System manifest info